README.md

```diff
@@ -1,75 +1,62 @@
+# [redux-simple-router](#docs)
```

Comment:
This link doesn't work right now.

(And I can't remember seeing other's link like this in the main headline, so I'm not sure we should do it. Maybe just remove the link? If it's too far down to the documentation, it just means we should remove some of the text above it)